executor/adapter.go

```diff
@@ -896,12 +937,130 @@ func FormatSQL(sql string) stringutil.StringerFunc {
 	}
 }
 
+const (
+	phaseBuildLocking       = "build:locking"
+	phaseOpenLocking        = "open:locking"
+	phaseNextLocking        = "next:locking"
+	phaseLockLocking        = "lock:locking"
+	phaseBuildFinal         = "build:final"
+	phaseOpenFinal          = "open:final"
+	phaseNextFinal          = "next:final"
+	phaseLockFinal          = "lock:final"
+	phaseCommitPrewrite     = "commit:prewrite"
+	phaseCommitCommit       = "commit:commit"
+	phaseCommitWaitCommitTS = "commit:wait:commit-ts"
+	phaseCommitWaitLatestTS = "commit:wait:latest-ts"
+	phaseCommitWaitLatch    = "commit:wait:local-latch"
+	phaseCommitWaitBinlog   = "commit:wait:prewrite-binlog"
+	phaseWriteResponse      = "write-response"
```

Comment:
@cfzjywxk @you06 @sticnarf @longfangsong Any suggestion about naming? eg. `build:locking` and `locking:build`, which is better?

Reply:
`build:locking` IMO, it's better to put the related metrics together in the alphabetic order.